### PR TITLE
Clarify Enabling the IRC Bot docs.

### PR DESCRIPTION
**Pull request by winksaville**

### CODE REVIEW

File: master/docs/tutorial/tour.rst [hunk 1]
```diff
@@ -166,13 +166,14 @@ First, start an IRC client of your choice, connect to irc.freenode.org and join
 In this example we will use #buildbot-test, so go join that channel.
 (*Note: please do not join the main buildbot channel!*)
 
-Edit the config and look for the *STATUS TARGETS* section.
-Enter these lines below the WebStatus line in :file:`master.cfg`::
+Edit :file:'master.cfg' and look for the *STATUS TARGETS* section. At the
+end of that section, just below the WebStatus line::
```
Comment (tardyp) [inline master/docs/tutorial/tour.rst]: well, actually, there is no such webstatus line.

File: master/docs/tutorial/tour.rst [hunk 2]
```diff
@@ -166,13 +166,14 @@ First, start an IRC client of your choice, connect to irc.freenode.org and join
 In this example we will use #buildbot-test, so go join that channel.
 (*Note: please do not join the main buildbot channel!*)
 
-Edit the config and look for the *STATUS TARGETS* section.
-Enter these lines below the WebStatus line in :file:`master.cfg`::
+Edit :file:'master.cfg' and look for the *STATUS TARGETS* section. At the
```
Comment (sa2ajj) [inline master/docs/tutorial/tour.rst]: One sentence per line please

File: master/docs/tutorial/tour.rst (hunk 1 above)
Comment (winksaville) [inline master/docs/tutorial/tour.rst]: You're correct! I now see that sample.cfg was changed here and that code was removed from sample.cfg. I was looking at the released version. I can reword but I wanted to test but I'm having trouble installing. I'll ask in the devel list.

File: master/docs/tutorial/tour.rst (hunk 2 above)
Comment (winksaville) [inline master/docs/tutorial/tour.rst]: Fixed.

File: master/docs/tutorial/tour.rst (hunk 1 above)
Comment (winksaville) [inline master/docs/tutorial/tour.rst]: Fixed.

**Comment by tardyp** [comment]
Thanks. I am going to fix the problem you are reporting in another PR

**Pull request merged**